db/fixtures/notification_types.yml

```diff
@@ -285,6 +285,11 @@
   :expires_in: 24.hours
   :level: :error
   :audience: superadmin
+- :name: enable_update_repo_failed
+  :message: 'Subscription Manager failed to enable repo %{repo_name}'
```

Comment:
I like how we're consistently inconsistent with putting periods at the end of the message.  🤣